database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -110,9 +110,25 @@ public void onChanged(EventType type, int index, int oldIndex) {
             public void onCancelled(DatabaseError databaseError) {
                 FirebaseRecyclerAdapter.this.onCancelled(databaseError);
             }
+
+            @Override
+            public void onReady() {
+                FirebaseRecyclerAdapter.this.onDataChanged();
```

Comment:
I think we should go for the second option (with `onChildChanged` letting us use `onDataChanged`). It would make the api more consistent with the native Firebase apis and fits well in the context of my other PR: https://github.com/firebase/FirebaseUI-Android/pull/488/files#diff-3620a1e9acaa17bea769cfbc3079f39aR18.